### PR TITLE
Clarify that the tournament committee doesn't operate the tournament email

### PR DESCRIPTION
**Pull request by Walavouchey**
was talked about in `#tournaments` a while back but i forgot to make the change <.<

### CODE REVIEW

**Comment by hburn7** [comment]
Thoughts on adding clarification to the people/tournament_committee section? Something like "Note: These users do not have access to the email tournaments@ppy.sh - this inbox is managed by the account support team". I think this current change is great but some extra clarification would be very welcome, especially for new users.

File: wiki/People/Tournament_Committee/en.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The panel was first announced to the public on July 22, 2021 through the `#tourn
 
 You can help the tournament committee by continuing to run excellent tournaments! If you have any issues or concerns regarding any aspect of a community tournament, we encourage you to bring these to our attention via the tournament reports form.
 
-Those who are interested in joining the tournament committee may express their interest to us by sending an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh). Successful candidates are expected to have a significant breadth of experience in the tournament scene, either as a player or staff.
+Those who are interested in joining the tournament committee may express their interest by sending an email to the [account support team](/wiki/People/Account_support_team) via [tournaments@ppy.sh](mailto:tournaments@ppy.sh). Successful candidates are expected to have a significant breadth of experience in the tournament scene, either as a player or staff.
```
Comment (Walavouchey) [inline wiki/People/Tournament_Committee/en.md:17]: @hburn7 this should be clear enough while being concise, as in there's no reason to conclude that the tournament committee is behind the email with this wording

```suggestion
Those who are interested in joining the tournament committee may express their interest by sending an email to the [account support team](/wiki/People/Account_support_team) via [tournaments@ppy.sh](mailto:tournaments@ppy.sh), where requests are forwarded to the committee. Successful candidates are expected to have a significant breadth of experience in the tournament scene, either as a player or staff.
```

Comment (hburn7) [reply wiki/People/Tournament_Committee/en.md:17]: Ah okay, yes I think this accomplishes that. Thanks!

**Review by TicClick — APPROVED** [review]

**Comment by TicClick** [comment]
(merge at your discretion)

**Pull request merged**